snarkos/config.rs

```diff
@@ -323,15 +321,19 @@ impl Config {
         }
     }
 
-    pub fn check(&self) -> Result<(), NetworkError> {
+    pub fn check(&self) -> Result<(), CliError> {
         // Check that the minimum and maximum number of peers is valid.
         if self.p2p.min_peers == 0 || self.p2p.max_peers == 0 {
-            return Err(NetworkError::PeerCountInvalid);
+            return Err(CliError::PeerCountInvalid);
         }
 
         // Check that the sync interval is a reasonable number of seconds.
         if !(2..=300).contains(&self.p2p.peer_sync_interval) || !(2..=300).contains(&self.p2p.block_sync_interval) {
-            return Err(NetworkError::SyncIntervalInvalid);
+            return Err(CliError::SyncIntervalInvalid);
```

Comment:
Note: we could parse instead of validating for all these intervals in future?

Reply:
We'll soon need to revise the synchronization intervals we're using right now, possibly with some other config cleanups, good point :+1:.